help.py

```diff
@@ -13,109 +13,78 @@ def __init__(self, attributes: dict):
-        return embed
 
+    # This function builds the text for the help (list) command (ex: y!help)
     async def build_list_bot_help(self, mapping: Mapping[Optional[commands.Cog], List[commands.Command]]) -> str:
-        prefix = os.getenv("PREFIX")
         result_string = 'Here is a list of all commands and their descriptions:\n\n'
+
+        # For every type (cog) of command (Utility, Info, Fun)
         for item in mapping.items():
-            result_string += '{} {}\n'.format(self.emoji_mapping[item[0].qualified_name], item[0].qualified_name) if item[0] is not None else '💡 No Category\n'
+            # If it has a type, add an entry for it
+            if item[0] is not None:
+                result_string += f'{self.emoji_mapping[item[0].qualified_name]} {item[0].qualified_name} \n'
+            else:
+                result_string += '💡 No Category\n'
+
+            # Get every command and their details from the cogs and then sort them
             cmds: List[commands.Command] = await self.filter_commands(commands=item[1], sort=True)
             deduplicated = sorted(set(cmds), key=lambda x: x.name)
+
+            # For every command; add them to the list
             for cmd in deduplicated:
-                result_string += '`{}` - {}\n'.format(cmd.name, cmd.description)
+                result_string += f'`{cmd.name}` - {cmd.description}\n'
             result_string += '\n'
         return result_string
 
-    def build_embed_cog_help(self, cog: commands.Cog) -> discord.Embed:
-        prefix = os.getenv("PREFIX")
-        author: Union[discord.User, discord.Member] = self.context.author
-        cmds: List[commands.Command] = cog.get_commands()
-        cog_name = cog.qualified_name
-        embed = discord.Embed(title=cog_name, description='Here is a list of commands for `' + cog_name + '`.',
-                              color=self.color)
-        embed.set_footer(text=f"Type {prefix}help <command> for more info on a command.")
-        embed.set_author(name=author.display_name, icon_url=author.avatar_url)
-        cmds = sorted(cmds, key=lambda x: x.name)
-        for cmd in cmds:
-            embed.add_field(name=cmd.name, value=cmd.description, inline=False)
-        return embed
-
+    # This function builds the text for the help [cog] command (ex: y!help Utility)
     def build_list_cog_help(self, cog: commands.Cog) -> str:
-        result_string = f'{self.emoji_mapping[cog.qualified_name]} Here is a list of commands for `{cog.qualified_name}`\n'
+        result_string = f'{self.emoji_mapping[cog.qualified_name]} Here is a list of commands in the `{cog.qualified_name}` category\n'
+
         cmds: List[commands.Command] = sorted(cog.get_commands(), key=lambda x: x.name)
         for cmd in cmds:
-            result_string += '`{}` - {}\n'.format(cmd.name, cmd.description)
+            result_string += f'`{cmd.name}` - {cmd.description}\n'
         return result_string
```

Comment:
I'm not sure if this feature is needed. All it really does is return a shorter version of the full help command. Usage: `y!help Utility` The other part of the code is on lines 82-85.